src/Components.js

```diff
@@ -69,7 +69,7 @@ export const BasicColHeader = styled.div`
   text-align: center;
   user-select: none;
   cursor: ${props => (props.sortable ? 'pointer' : 'default')};
-  background-color: ${props => props.backgroundColor || 'steelblue'};
+  background-color: ${props => props.backgroundColor};
   color: ${props => props.color || 'white'};
```

Comment:
we would want to control the color as well as the font-size and font-weight props as well

Reply:
Sure, would add those.